openapi-cli/src/test/resources/ballerina-to-openapi/constraint/array.bal

```diff
@@ -18,7 +18,7 @@ public type Hobby HobbyItemsString[];
 
 public type Person record {
     Hobby hobby?;
-    @constraint:Array {maxLength: 5}
+    @constraint:Array {length: 15}
```

Comment:
Shall we add a separate field to capture the `length` constraint instead of changing the existing `maxlength` field?

Reply:
Done